mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -102,6 +129,36 @@ static FailureOr<Operation *> getCompressedMaskOp(OpBuilder &rewriter,
   return newMask;
 }
 
+static std::optional<int64_t>
+getIntraVectorOffset(ConversionPatternRewriter &rewriter, Location loc,
+                     const memref::LinearizedMemRefInfo linearizedInfo) {
+  auto foldedIntraVectorOffset = getValueOrCreateConstantIndexOp(
+      rewriter, loc, linearizedInfo.intraVectorOffset);
+  return getConstantIntValue(foldedIntraVectorOffset);
+}
+
+static Value extractSubvectorFrom(ConversionPatternRewriter &rewriter,
```

Comment:
I thought that I made a suggestion to use `RewriteBase&`, but it looks like I missed it. In the codebase we usually pass `RewriteBase&` to a method because the method can be reused by any other rewriters. There are no ConversionPatternRewriter specific methods needed in the function. It helps the case that we could refactor it to somewhere else and the consistency in the codebase. Can you make the change? (Same for the below function.)